assets/src/edit-story/components/canvas/displayElement.js

```diff
@@ -51,13 +52,38 @@ const BackgroundOverlay = styled.div`
   top: 0;
   left: 0;
 `;
+
 const ReplacementContainer = styled.div`
   transition: opacity 0.25s cubic-bezier(0, 0, 0.54, 1);
   pointer-events: none;
   opacity: ${({ hasReplacement }) => (hasReplacement ? 1 : 0)};
 `;
 
-function DisplayElement({ element, previewMode, page }) {
+function AnimationWrapper({ children, id, animationMode }) {
+  switch (animationMode) {
+    case 'WAAPI':
+      return (
+        <StoryAnimation.WAAPIWrapper target={id}>
+          {children}
+        </StoryAnimation.WAAPIWrapper>
+      );
+    case 'AMP':
+    default:
+      return children;
+  }
+}
+AnimationWrapper.propTypes = {
+  animationMode: PropTypes.oneOf(['none', 'WAAPI', 'AMP']).isRequired,
```

Comment:
Nit: Could also make these a constant in constants/animation and easily reference them where you need them. I know they are weird things to misspell but it might be nice.

Reply:
yea I think I'm weening myself off raw strings slowly since they're super safe in TS, but not so much in JS. bad habit I need to break